keep-ui/app/ai/ai.tsx

```diff
@@ -1,19 +1,27 @@
 "use client";
 import { Card, List, ListItem, Title, Subtitle } from "@tremor/react";
-import { useAIStats } from "utils/hooks/useAIStats";
+import { useAIStats } from "utils/hooks/useAI";
 import { useSession } from "next-auth/react";
 import { getApiURL } from "utils/apiUrl";
 import { toast } from "react-toastify";
 import { useEffect, useState, useRef, FormEvent } from "react";
+import { usePollAILogs } from "utils/hooks/useAI";
+import { DialogOverlayProps } from "@headlessui/react";
 
 export default function Ai() {
   const { data: aistats, isLoading } = useAIStats();
   const { data: session } = useSession();
   const [text, setText] = useState("");
+  const [basicAlgorithmLog, setBasicAlgorithmLog] = useState("");
   const [newText, setNewText] = useState("Mine incidents");
   const [animate, setAnimate] = useState(false);
   const onlyOnce = useRef(false);
 
+  const mutateAILogs = (logs: any) => {
```

Comment:
any is bad practice, if we have some type for it

Reply:
<3